docs/markdown/Subprojects.md

```diff
@@ -175,9 +176,22 @@ executable('my_project',
   install : true)
 ```
 
-With this setup when libsimple is provided by the system, we use it.
-When that is not the case we use the embedded version (the one from
-subprojects).
```

Comment:
It's not clear to me why you dropped the last sentence here.

Reply:
It was unintentional, I was working too late at night again. ;) Fixed (afaik)

Reply:
Thanks, I was wondering if I had missed something!